features/app_update.feature

```diff
@@ -1,6 +1,7 @@
 Feature: App update test
 
-@Integration
+# Not AWS due to airplane mode
```

Comment:
@amstone326 why did you add this comment back in? Its not true right? (IE we can disable the internet on AWS)